app/views/helpers/feed/update.phtml

```diff
@@ -846,7 +846,7 @@
 						$httpHeaders = array_filter($httpHeaders,
 							fn(string $header) => !preg_match('/^(Remote-User|X-WebAuth-User)\\s*:/i', $header));
 					?>
-					<textarea class="valid-json" id="http_headers" name="http_headers" rows="3" cols="64" spellcheck="false"><?php
+					<textarea class="w100" id="http_headers" name="http_headers" rows="3" spellcheck="false"><?php
```

Comment:
Strangely that `.valid-json` doesn't even seem to be used anywhere in JS or CSS? It's just on a few textareas.

Reply:
yes, I saw it too. But I did not wanted to go deeper to investigate the reasons

Reply:
This was implemented with #5662
Ping @eta-orionis

Reply:
From memory, this was to mimic `valid-xpath` which has some auto-validation logic, but that has not been implemented for JSON yet